CMakeLists.txt

```diff
@@ -223,37 +296,27 @@ endif()
 
 # Some of the external libraries are not used for mobile.
 if(DESKTOP)
-  # Use the static versions of the OpenSSL libraries.
-  set(OPENSSL_USE_STATIC_LIBS TRUE)
-  if (MSVC)
-    # Get the correct version of the OpenSSL libraries based on building for MT.
-    if ("${CMAKE_CXX_FLAGS_DEBUG}" MATCHES "/MT" OR
-        "${CMAKE_CXX_FLAGS_RELEASE}" MATCHES "/MT")
-      set(OPENSSL_MSVC_STATIC_RT TRUE)
-    else()
-      set(OPENSSL_MSVC_STATIC_RT FALSE)
-    endif()
-  endif()
-
   # Build curl as a static library
   set(CURL_STATICLIB ON CACHE BOOL "")
   if (WIN32)
-    set(CMAKE_USE_WINSSL ON CACHE BOOL "")
+    set(CMAKE_USE_SCHANNEL ON CACHE BOOL "")
```

Comment:
nit: will be great to put a one line comment explaining what this option does,
"enable Windows native SSL/TLS"